File: Python3/1013 - O Maior.py
a, b, c = map(int, input().split())
MaiorAB = (a + b + abs(a - b)) / 2
Resultado = (MaiorAB + c + abs(MaiorAB - c)) / 2
print('{:.0f} eh o maior'.format(Resultado))

File: C#/1013 - O Maior.cs
using System;

class MainClass {
  public static void Main (string[] args) {
    string[] S = Console.ReadLine().Split();
    int A = int.Parse(S[0]);
    int B = int.Parse(S[1]);
    int C = int.Parse(S[2]);
    if (A >= B && A >= C)
      Console.WriteLine(A + " eh o maior");
    else if (B >= A && B >= C)
      Console.WriteLine(B + " eh o maior");
    else if (C >= A && C >= B)
      Console.WriteLine(C + " eh o maior");
  }
}